plugins/macvlan.go

```diff
@@ -0,0 +1,43 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugins
+
+const (
+	// Macvlan specifies macvlan network
+	Macvlan = "macvlan"
+	// DefaultIpamForMacvlan specifies the default ipam type for macvlan
+	DefaultIpamForMacvlan = "dhcp"
```

Comment:
@satyaranjanp 
Since it is likely that dhcp is not present on the host, should you not change the default to "host-local" and also provide a default "subnet" ?
Is the presence of dhcp expected for the normal operation of macvlan?